src/aap_eda/tasks/orchestrator.py

```diff
@@ -495,3 +480,12 @@ def monitor_rulebook_processes() -> None:
             process_parent_id,
             None,
         )
+
+
+def monitor_rulebook_processes_unique() -> None:
```

Comment:
```suggestion
def enqueue_monitor_rulebook_processes() -> None:
```

This naming expresses the functional aspect rather than the implementation.

Reply:
changed. Thanks